### DISCUSSION TITLE
Control lips, teeth and tongue for lip sync task.

### DISCUSSION THREAD

**Discussion by tumurzakov** [lllyasviel/ControlNet#366] [Ideas]
I want precisely control mouth for stylize video. There is model for controlnet ControlNetMediaPipeFace from CrucibleAI. But there is no enought control for lips, teeth and tongue.

1. Is there a way to make lips sync without training special model for control net?
2. Is it possible to train extension for ControlNetMediaPipeFace to make lips sync?

[Discussion on huggingface](https://huggingface.co/CrucibleAI/ControlNetMediaPipeFace/discussions/20)
[Redit post](https://www.reddit.com/r/ControlNet/comments/12xq9rb/is_there_a_way_to_control_lips_teeth_and_tongue/)

![изображение](https://user-images.githubusercontent.com/440364/234075574-051bdf5d-631b-483b-8f52-ac262646fca9.png)

**Comment by geroldmeisinger** [comment]
ControlNet 1.1 has support for faces now https://github.com/lllyasviel/ControlNet-v1-1-nightly#controlnet-11-openpose

**Comment by johndpope** [comment]
@tumurzakov - did you use the ControlNet 1.1 implementation? how did you find it?  i just spent an hour wiring up the ControlNetMediaPipeFace
